### ISSUE TITLE
support C stubs

### ISSUE THREAD

**Issue by agarwal** [solvuu/solvuu_build#6]
Would be good to use `solvuu_build` for Biocaml, but we can't right now since Biocaml has some C stubs.

**Comment by agarwal** [comment]
It seems @pveber did do this at some point, but I probably broke it again. As of right now, I'm mistakenly doing the identical thing in both the "use_foo_stub for apps” and the “use_foo for apps” parts of generating the tags file. This likely happened during my mass refactoring.

**Comment by agarwal** [comment]
Partly done in 241cb4f2ba1ea8cab87644b7e90b3b1748c21bdb, but all kinds of problems remain.

**Comment by pveber** [comment]
Just as a memo: in `biocaml.unix`, there was a module using C stubs. The corresponding unit test program would not run saying it couldn't find `dllbiocaml_unix.so`. Sorry the description might not be very useful but this is just a reminder that it is worth checking byte executable using C stubs do work when compiled using `solvuu-build`.

**Comment by smondet** [comment]
@pveber for running bytecode tests that need a `.so` you may just need to set `LD_LIBRARY_PATH` (at least on Linux) (once the library is installed findlib takes care of that, cf. ``  `ocamlc -where`/ld.conf ``)

See https://gitlab.com/smondet/misuja/blob/master/README.md#L25-37
(I didn't use solvuu-build for that project though; it's based on `ocamlmklib`; it's a mostly-C library with a small user-facing OCaml API).

**Comment by pveber** [comment]
Right, silly me. I thought I had tested that, but since you were mentioning it I decided to give it a new try, and of course it works... Sorry for the noise!
